Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -239,7 +240,9 @@ namespace Babylon
             }
         }
 
-        m_session = std::make_unique<xr::System::Session>(m_system, bgfx::getInternalData()->context);
+        return xr::System::Session::CreateAsync(runtimeScheduler, m_system, bgfx::getInternalData()->context).then(runtimeScheduler, arcana::cancellation::none(), [this](std::shared_ptr<xr::System::Session> session) {
```

Comment:
Since the final continuation is just doing an assignment, can it just use `inline_scheduler`?